### PR TITLE
Fixes #3185 to bring install and local docs inline.

### LINKED ISSUE #3185: System Requirements Docs Issues
Currently our system requirements / installation docs are out of sync. 

We should be consistent, and have the correct php versions in the docs.

### PR DESCRIPTION
**Pull request by mikemadison13**
Fixes #3185 
--------

Changes proposed:
---------
(What are you proposing we change?)

- updates php requirements in install docs to be php71
- removes specific drupal vm requirements from install docs and instead links to drupal vm docs (currently install and local docs are not in sync, ansible was missing from install docs, etc.)
- adds drush as a minimum dependency for blt

### CODE REVIEW

**Comment by mikemadison13** [comment]
@geerlingguy while we are updating these, could you please have a look to make sure that we aren't overlooking any minimum requirements for drupal vm, or any other recommended plugins?

File: docs/INSTALL.md [hunk 1]
```diff
@@ -30,16 +30,12 @@ Ensure that [Xcode](https://itunes.apple.com/us/app/xcode/id497799835?mt=12) is
 Then install the minimum dependencies for BLT. The preferred method is via Homebrew, though you could install these yourself without a package manager.
 
         /usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
-        brew install php71 git composer
+        brew install php71 git composer drush
```
Comment (geerlingguy) [inline docs/INSTALL.md]: Is Drush an actual requirement, though? I thought it was only necessary if you want to run Drush commands outside the VM.

File: docs/local-development.md [hunk 2]
```diff
@@ -6,7 +6,7 @@ Acquia currently recommends the use of either:
   * [Drupal VM](#using-drupal-vm-for-blt-generated-projects): an isolated virtual machine, built with Vagrant and Ansible
   * [Acquia Dev Desktop](#using-acquia-dev-desktop-for-blt-generated-projects): a turn-key LAMP stack tailored specifically for Acquia-hosted Drupal sites
   * [Alternative local development environments](#alternative-local-development-environments)
-
+
```
Comment (geerlingguy) [inline docs/local-development.md]: Extra space needs to go away.

File: docs/local-development.md [hunk 3]
```diff
@@ -21,8 +21,9 @@ To use [Drupal VM](http://www.drupalvm.com/) with a Drupal project that is gener
 1. Download the Drupal VM dependencies listed in [Drupal VM's README](https://github.com/geerlingguy/drupal-vm#quick-start-guide). If you're running [Homebrew](http://brew.sh/index.html) on Mac OSX, this is as simple as:
```
Comment (geerlingguy) [inline docs/local-development.md]: s/Mac OSX/macOS

File: docs/local-development.md [hunk 4]
```diff
@@ -21,8 +21,9 @@ To use [Drupal VM](http://www.drupalvm.com/) with a Drupal project that is gener
...
 
         brew tap caskroom/cask
-        brew install php56 git composer ansible drush
+        brew install php71 git composer ansible drush
         brew cask install virtualbox vagrant
+        vagrant plugin install vagrant-hostsupdater
```
Comment (geerlingguy) [inline docs/local-development.md]: This is done automatically by Drupal VM, no need to add a separate command here.

**Comment by mikemadison13** [comment]
thanks @geerlingguy  will update!

**Pull request merged**